src/prefect/tasks/dbt/dbt.py

```diff
@@ -262,6 +263,7 @@ def __init__(
         token_env_var_name: str = "DBT_CLOUD_TOKEN",
         wait_for_job_run_completion: bool = False,
         max_wait_time: int = None,
+        domain: str = None,
```

Comment:
I recommend setting the default value of this parameter to `cloud.getdbt.com` like so:
```suggestion
        domain: str = "cloud.getdbt.com",
```